src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/ComEventsHelperTests.cs

```diff
@@ -11,10 +11,18 @@ public class ComEventsHelperTests
     {
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
         public void Combine_NullRcwWindows_ThrowsArgumentNullException()
         {
             AssertExtensions.Throws<ArgumentNullException>(null, () => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
+        public void Combine_NullRcwUapAot_PlatformNotSupportedException()
```

Comment:
What we could do is instead of adding new tests that expect PNSE, then only disable the test against one issue that points we need to re-enable them and do the change in coreRT. When coreRT change is ingested we can then re-enable the tests. 

That way we will match coreCLR and coreRT behaviors and also clean the failures in the meantime.